jgrapht-ext/src/test/java/org/jgrapht/ext/DOTExporterTest.java

```diff
@@ -43,8 +43,10 @@
 
     // TODO jvs 23-Dec-2006: externalized diff-based testing framework
 
-    private static final String UNDIRECTED = "graph G {" + NL + "  1 [ label=\"a\" ];" + NL
-        + "  2 [ x=\"y\" ];" + NL + "  3;" + NL + "  1 -- 2;" + NL + "  3 -- 1;" + NL + "}" + NL;
+    private static final String UNDIRECTED = DOTExporter.DONT_ALLOW_MULTIPLE_EDGES_KEYWORD
```

Comment:
I always prefer to keep in sync the tests with the static keywords, because if you change one of them you know that the compatibility with previous versions is broken. But I changed it anyway.